credsweeper/filters/value_jfrog_token_check.py

```diff
@@ -28,10 +31,19 @@ def run(self, line_data: LineData, target: AnalysisTarget) -> bool:
             True, if need to filter candidate and False if left
 
         """
-        if not line_data.value:
+        value = line_data.value
+        if not value:
             return True
         with contextlib.suppress(Exception):
-            decoded = Util.decode_base64(line_data.value, padding_safe=True, urlsafe_detect=True)
-            if self._pattern.match(decoded.decode(ASCII)):
-                return False
+            if value.startswith("cmVmdGtuO"):
+                decoded = Util.decode_base64(value, padding_safe=True, urlsafe_detect=True)
+                if self._pattern.match(decoded.decode(ASCII)):
+                    # identity token
+                    return False
+            if value.startswith("AKCp"):
+                decoded = base58.b58decode(value)
+                # the check only for correct size decoding
+                if 54 == len(decoded):
+                    # API key (deprecated) - a good integrity check solution was not found
```

Comment:
Could you please explain why `54` is used for the check?
Following your mail there is only limited information on link.
AKCp2UNCd8uK7hQoxZnFE4PGtRHnAcBHr43HgLcj7nJmWb4JhVUqBwa2iwXszftnogpo2EVFa

There are some prefixes and minimal lengths, but nothing about `54`..

Reply:
54 is length of decoded bytes.  All tokens i found (except obviously test-like) are base58 encoding. 
There is a lite "validation" in case of wrong encoding - an exception filters the value

Reply:
Hmm... I can't understand enough...
Each tokens with BASE58 encoding are having more than 73 length and it means decoded strings can have various length.. isn't it?

Reply:
The token type has 73 symbols length which decodes to 54 bytes via base58 (base64 is not acceptable due left 4 bits). 
So, my suggestion is the base58 decoding should produce 54 bytes from the token (experience from found tokens).

Reply:
Yup BASE58 string which has 73 length is decoded to 54 bytes, but there are not only 73 length of tokens as you can see the JFrog code, it is more than..
So what about change the if condition to `<=`?

Reply:
As soon the API key is deprecated, IMO the check was designed for an extension, but it will not realized. So, the size of token now used is exactly 73. bigger size may produce false positives in many base64 encodings. "AKCp" is a short signature unfortunately.

Reply:
Okay. I got it.
Thank you.